qiskit/quantum_info/synthesis/one_qubit_decompose.py

```diff
@@ -220,59 +219,21 @@ def angles_and_phase(self, unitary):
     @staticmethod
     def _params_zyz(mat):
         """Return the Euler angles and phase for the ZYZ basis."""
-        # We rescale the input matrix to be special unitary (det(U) = 1)
-        # This ensures that the quaternion representation is real
-        coeff = np.linalg.det(mat) ** (-0.5)
-        phase = -cmath.phase(coeff)
-        su_mat = coeff * mat  # U in SU(2)
-        # OpenQASM SU(2) parameterization:
-        # U[0, 0] = exp(-i(phi+lambda)/2) * cos(theta/2)
-        # U[0, 1] = -exp(-i(phi-lambda)/2) * sin(theta/2)
-        # U[1, 0] = exp(i(phi-lambda)/2) * sin(theta/2)
-        # U[1, 1] = exp(i(phi+lambda)/2) * cos(theta/2)
-        theta = 2 * math.atan2(abs(su_mat[1, 0]), abs(su_mat[0, 0]))
-        phiplambda2 = cmath.phase(su_mat[1, 1])
-        phimlambda2 = cmath.phase(su_mat[1, 0])
-        phi = phiplambda2 + phimlambda2
-        lam = phiplambda2 - phimlambda2
-        return theta, phi, lam, phase
+        return euler_one_qubit_decomposer.params_zyz(mat)
```

Comment:
I don't know if it'll have much effect, but at this point we may just want to bind
```python
_params_zyz = staticmethod(euler_one_qubit_decomposer.params_zyz)
```
(and so on) to avoid an unnecessary Python function-call overhead.  If it doesn't produce a meaningful speedup, though, I wouldn't bother.

Reply:
This is a good idea I'll give this a try. It probably won't be super noticeable but it won't make it any slower, a python function ends up being like ~100ns so removing one layer can't hurt

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9185/commits/3aeea03be983c4ec16745a4fdaed23de2db9549f